front/src/routes/chat/style.css

```
@@ -574,6 +579,69 @@
+}
+
+.voiceInputButtonListening,
+.voiceInputButtonListening:hover,
+.voiceInputButtonListening:focus {
+  background: #e74c3c;
+  color: #fff;
+  box-shadow: 0 2px 8px rgba(231, 76, 60, 0.4);
+  animation: voiceInputPulse 1.6s ease-in-out infinite;
+}
+
+@keyframes voiceInputPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "voiceInputPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)